src/lib/phdi-transforms/phdi_transforms/geo.py

```diff
@@ -0,0 +1,59 @@
+from typing import TypedDict, List
+
+from smartystreets_python_sdk import StaticCredentials, ClientBuilder
+from smartystreets_python_sdk import us_street
+from smartystreets_python_sdk.us_street.lookup import Lookup
+
+
+class GeocodeResult(TypedDict):
+    """
+    A class representing a successful geocoding response
+    """
+
+    key: str
+    address: List[str]
+    zipcode: str
+    fips: str
+    lat: float
+    lng: float
+
+
+def geocode(client: us_street.Client, address: str) -> GeocodeResult:
+    """
+    Given an API client and an address, attempt to call the smartystreets API
+    """
+
+    lookup = Lookup(street=address)
+    client.send_lookup(lookup)
+
+    if lookup.result:
+        res = lookup.result[0]
+
+        addr = [res.delivery_line_1]
+        if res.delivery_line_2:
+            addr.append(res.delivery_line_2)
+
+        return {
```

Comment:
`city` and `state` are included in the return, but are not defined in the `GeocodeResult` class, and additionally `key` is defined in the class but not included in the output. Does this matter?